### PR TITLE
WMImplant detection rule

### PR DESCRIPTION
**Pull request by SpeedyFireCyclone**
Rule to detect CLI parameters used by WMImplant.

### CODE REVIEW

**Comment by Neo23x0** [comment]
Some strings looks as if they'd be prone to false positives. I cannot make changes to that file (don't know why).

**Comment by SpeedyFireCyclone** [comment]
I know they seem like that at first glance, but they've not actually triggered any false positives for us yet.
Perhaps I could change the false positive property to unknown instead of unlikely?

**Comment by Neo23x0** [comment]
The false positives section doesn't work that way. We use it to describe situations or tools that could possibly lead to false positives. 
E.g. in your case I'd write
```
    - Administrative scripts, which just happen to use the same keywords used by WMIimplant
```

File: rules/windows/powershell/powershell_wmimplant.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - " disable_wdigest "
+      - " disable_winrm "
+      - " enable_wdigest "
+      - " enable_winrm "
+      - " registry_mod "
+      - " remote_posh "
+      - " sched_job "
+      - " service_mod "
+      - " process_kill "
+      - " process_start "
```
Comment (Neo23x0) [inline rules/windows/powershell/powershell_wmimplant.yml]: ^ please disable with comment

File: rules/windows/powershell/powershell_wmimplant.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - " active_users "
+      - " basic_info "
+      - " drive_list "
```
Comment (Neo23x0) [inline rules/windows/powershell/powershell_wmimplant.yml]: ^ please disable with comment

File: rules/windows/powershell/powershell_wmimplant.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      - " installed_programs "
```
Comment (Neo23x0) [inline rules/windows/powershell/powershell_wmimplant.yml]: ^ please disable with comment

File: rules/windows/powershell/powershell_wmimplant.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      - " power_off "
+      - " vacant_system "
+      - " logon_events "
+  condition: selection
+falsepositives:
+  - unlikely
```
Comment (Neo23x0) [inline rules/windows/powershell/powershell_wmimplant.yml]: change to comment similar to the one I gave you in the "Conversation" tab

**Comment by SpeedyFireCyclone** [comment]
@Neo23x0, I fixed the points you brought up, could it get merged?

**Pull request merged**